packages/slots/SlotRenderer.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import config from '@plone/registry';
+
+import type { GetSlotArgs } from '@plone/types';
+
+/*
+ * The SlotRenderer component is used to render the components registered for a
+ * given slot.
+ *
+ * Usage:
+ * <SlotRenderer name="aboveContent" content={content} location={location} />
+ */
+const SlotRenderer = ({
+  name,
+  content,
+  location,
+  navRoot,
+}: {
+  name: string;
+  content: GetSlotArgs['content'];
+  location: GetSlotArgs['location'];
+  navRoot?: GetSlotArgs['navRoot'];
+}) => {
+  let slots = config.getSlot(name, {
+    content,
+    location,
+    // This is to cover the use case while adding a new content and we don't have
```

Comment:
"have" is repeated on the next line

```suggestion
    // This is to cover the use case while adding a new content and we don't
```